authcrypt/main.cpp

```diff
@@ -25,16 +25,18 @@
 
 int main() {
     mbedtls_platform_context platform_ctx;
-    if(mbedtls_platform_setup(&platform_ctx) != 0) {
-        return -1;
+    int exit_code = MBEDTLS_EXIT_FAILURE;
```

Comment:
As far as I can tell, regardless of whther mbedtls_platform_setup and run return, MBEDTLS_EXIT_FAILURE will always be returned. Is this correct?

Reply:
No, the value will be overriden by platform_setup and authcrypt->run.